### ISSUE TITLE
#7274 [mB] replace ck editor - If uploading image, alt-text and copyright-infos are not mandatory 

### ISSUE THREAD

**Issue by pascalkienast** [liqd/a4-meinberlin#5394]
**URL:** https://meinberlin-dev.liqd.net
**user:** all user who can upload in ck editor
**expected behaviour:** as defined in the userstory #7274: "alt text and copyright fields need to be mandatory for uploads"
**behaviour:** if i upload an image you will not be asked again to insert an alt text or image caption (which would then be the copyright text) and can then simply publish
**important screensize:** any
**device & browser:** any
**Comment/Question:** Since the upload does not work if you are initiator, I can only test this as an admin

Screenshot?

**Comment by goapunk** [comment]
@CarolingerSeilchenspringer I'm not sure what to do here. Making the alt text mandatory will be quite difficult. The same goes for the copyright (as @pascalkienast already said the only way would be to use the image caption for that or write a new ckeditor plugin which integrates a copyright, which would be a lot of work). As the old ckeditor also doesn't have mandatory alt text or copyright, do we absolutely have to do this? if so we will need some big stories for that

**Comment by goapunk** [comment]
@CarolingerSeilchenspringer ok, I might have found an acceptable way of enforcing the alt text. Will test it a bit more and maybe we can talk about it then

**Comment by CarolingerSeilchenspringer** [comment]
@goapunk here is the text for the missing alt text alert: Please add an alternative text for all images. (german: Bitte fügen Sie für alle Bilder einen Alternativtext hinzu.)
